HenFwork.MapEditing.VisualTests/Screens/MapSelect/MapSelectScreenTestScene.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Affectionate Dove <contact@affectionatedove.com>.
+// Licensed under the Affectionate Dove Limited Code Viewing License.
+// See the LICENSE file in the repository root for full license text.
+
+using HenFwork.MapEditing.Saves;
+using HenFwork.MapEditing.Screens.MapSelect;
+using HenFwork.Testing;
+using System;
+using System.Collections.Generic;
+
+namespace HenFwork.MapEditing.VisualTests.Screens.MapSelect
+{
+    public class MapSelectScreenTestScene : VisualTestScene
+    {
+        public List<Saves.WorldSave> worldSaves = new List<Saves.WorldSave>();
```

Comment:
Niepotrzebne jest deklarować to w zasięgu klasy, lepiej przenieść do konstruktora jako lokalną zmienną.